### DISCUSSION TITLE
RobustScanner - size mismatch for decoder.hybrid_decoder.embedding.weight

### DISCUSSION THREAD

**Discussion by Vania-Guima** [open-mmlab/mmocr#1800] [General]
I downloaded your pre-trained RobustScanner model. Doing the inference in the Unitail dataset, the following warning appears:

size mismatch for decoder.hybrid_decoder.embedding.weight: copying a param with shape torch.Size([93, 128]) from checkpoint, the shape in current model is torch.Size([92, 128]).
size mismatch for decoder.prediction.weight: copying a param with shape torch.Size([93, 512]) from checkpoint, the shape in current model is torch.Size([92, 512]).
size mismatch for decoder.prediction.bias: copying a param with shape torch.Size([93]) from checkpoint, the shape in current model is torch.Size([92]).

Comparing your log file and my configuration file, all the parameters are the same. Where can I change the size of decoder?

**Comment by gaotongxiao** [comment]
Seems like you have modified the dictionary file, it affects the model's parameter as well.

**Reply by Vania-Guima** [reply]
You have right. I had 'unknown_token=None' from another script. Thanks a lot.
